### PR TITLE
Hotfixes

### PR DESCRIPTION
**Pull request by josemigallas**
Fixes some typos and adds a "pretest" script that runs eslint client.

### CODE REVIEW

**Comment by matzew** [comment]
:+1:

**Comment by matzew** [comment]
tested it - works,

but looks like there is a test failure

**Comment by josemigallas** [comment]
All typos fixed and removed some wrong fields from Message / Options models.

File: spec/util/test-runner-buider.spec.js [hunk 1]
```diff
@@ -57,15 +57,15 @@ describe("TestRunnerBuilder", () => {
...
-            const testRunners = testRunnerBuilder.buildTesstRunners();
+            const testRunners = testRunnerBuilder.buildTestRunners();
 
             expect(Array.isArray(testRunners)).toBeTruthy();
         });
 
         it("should create an array with as many TestRunner as 'instances' value", () => {
             const instances = 3;
             args.instances = instances;
-            const testRunners = testRunnerBuilder.buildTesstRunners();
```
Comment (matzew) [inline spec/util/test-runner-buider.spec.js:68]: :jack_o_lantern:

Comment (matzew) [reply spec/util/test-runner-buider.spec.js:68]: LOL

**Pull request merged**